templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+     <li>User: {{ user.get_username }}</li>
```

Comment:
Ensure that `user.get_username` is called as a method with parentheses: `user.get_username()`. This will correctly display the username.